avm/res/network/virtual-network-gateway/main.bicep

```
@@ -1,6 +1,47 @@
 metadata name = 'Virtual Network Gateways'
 metadata description = 'This module deploys a Virtual Network Gateway.'
 
+// ============= //
+// User Types    //
+// ============= //
+
+@description('Configuration for Virtual Network Gateway autoscale bounds.')
```

UDTs should be exported for reusability. https://azure.github.io/Azure-Verified-Modules/spec/BCPNFR20/

```suggestion
@export()
@description('Configuration for Virtual Network Gateway autoscale bounds.')
```